src/plots/gl3d/scene.js

```diff
@@ -708,4 +705,11 @@ proto.toImage = function(format) {
     return dataURL;
 };
 
+proto.setConvert = function() {
+    for(var i = 0; i < 3; ++i) {
+        var ax = this.fullSceneLayout[axisProperties[i]];
+        Axes.setConvert(ax, this.fullLayout);
+    }
+};
```

Comment:
Nice simplification! Why don't we need the [`containerOut.setScale = Lib.noop;`](https://github.com/plotly/plotly.js/pull/1431/files#diff-dd23546f2ee4a55150cd584d026e1c93L18) line anymore?